pkg/cli/dump_test.go

```diff
@@ -858,3 +860,171 @@ INSERT INTO foo (id) VALUES
 		})
 	}
 }
+
+func TestDumpTempTables(t *testing.T) {
```

Comment:
use the datadriven dump tests instead -- take a look at `pkg/cli/testdata/dump/`